src/libraries/System.Diagnostics.TraceSource/src/System/Diagnostics/Trace.cs

```diff
@@ -159,8 +159,18 @@ public static void Fail(string? message, string? detailMessage)
             TraceInternal.Fail(message, detailMessage);
         }
 
+        /// <summary>
+        ///  Occurs when a <see cref="TraceSource"/> needs to be refreshed from configuration.
+        /// </summary>
+        public static event EventHandler? Refreshing;
+        internal static void OnRefreshing()
```

Comment:
[nit] Missing blank line in between. Not sure what the policy for this in this repo. 
Ditto below

Reply:
Thanks. Done.